### PR TITLE
feat(tier4_planning_rviz_plugins): add vehicle_info to Footprint Displays

### LINKED ISSUE #710: PathFootprint/Trajectory Footprint's size is not the same as the vehicle size.
### Checklist

- [X] I've read the [contribution guidelines](https://github.com/autowarefoundation/autoware/blob/main/CONTRIBUTING.md).
- [X] I've searched other issues and no duplicate issues were found.
- [X] I'm convinced that this is not my fault but a bug.

### Description

Currently, path footprint/trajectory footprint's size is a constant value, which should be the vehicle's size from the vehicle_info package.
https://github.com/autowarefoundation/autoware.universe/blob/1e829e2948222e8b9c60a2af3bf3008ffbe2ec6d/common/tier4_planning_rviz_plugin/src/path_footprint/display.cpp#L35-L40

### Expected behavior

Path footprint/trajectory footprint's size will be the vehicle's size.

### Actual behavior

Path footprint/trajectory footprint's size is a constant value.

### Steps to reproduce

visualize path footprint or trajectory footprint plugin

### Versions

_No response_

### Possible causes

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by kenji-miyake**
## Description

Resolves #710 

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kenji-miyake** [comment]
@takayuki5168 Could you update the code as you like?

**Comment by kenji-miyake** [comment]
If you launch RViz separately from Autoware, you have to prepare a launch file that will load Autoware's global parameters and run RViz.

**Comment by takayuki5168** [comment]
Ah now I think I got it.
I understand that in rviz plugin we just get vehicle parameters via the rviz ros node if we launch the plugin with autoware.

I'll update this PR. Thank you so much!

**Comment by takayuki5168** [comment]
@kenji-miyake I updated all other plugins. I've already verified that all of them worked well. Give me LGTM.

**Comment by takayuki5168** [comment]
Ahh wait. This is your PR. So if you think it's ok to merge, let me know. Then I will approve and merge.

File: common/tier4_planning_rviz_plugin/src/path_footprint/display.cpp [hunk 1]
```diff
@@ -176,7 +190,13 @@ void AutowarePathFootprintDisplay::updateVehicleInfo()
   float width{property_vehicle_width_->getFloat()};
   float rear_overhang{property_rear_overhang_->getFloat()};
 
-  vehicle_footprint_info_ = std::make_shared<VehicleFootprintInfo>(length, width, rear_overhang);
+  if (vehicle_info_) {
+    vehicle_footprint_info_ = std::make_shared<VehicleFootprintInfo>(
+      vehicle_info_->vehicle_length_m, vehicle_info_->vehicle_width_m,
+      vehicle_info_->rear_overhang_m);
+  } else {
+    vehicle_footprint_info_ = std::make_shared<VehicleFootprintInfo>(length, width, rear_overhang);
```
Comment (kenji-miyake) [inline common/tier4_planning_rviz_plugin/src/path_footprint/display.cpp:198]: @takayuki5168 I'm sorry, but I should have minimized the scope. What do you think about it?

```suggestion
    const float length{property_vehicle_length_->getFloat()};
    const float width{property_vehicle_width_->getFloat()};
    const float rear_overhang{property_rear_overhang_->getFloat()};
    vehicle_footprint_info_ = std::make_shared<VehicleFootprintInfo>(length, width, rear_overhang);
```

Comment (kenji-miyake) [reply common/tier4_planning_rviz_plugin/src/path_footprint/display.cpp:198]: (I personally feel we can even delete these local variables.)

Comment (takayuki5168) [reply common/tier4_planning_rviz_plugin/src/path_footprint/display.cpp:198]: I feel the same. I'll update.

Comment (takayuki5168) [reply common/tier4_planning_rviz_plugin/src/path_footprint/display.cpp:198]: about minimizing the scope

Comment (kenji-miyake) [reply common/tier4_planning_rviz_plugin/src/path_footprint/display.cpp:198]: After that, I think you can merge this!

Comment (takayuki5168) [reply common/tier4_planning_rviz_plugin/src/path_footprint/display.cpp:198]: https://github.com/autowarefoundation/autoware.universe/pull/712/commits/c913b4610571e2f6ec46929dc9023a2c82cb26af
Done. Thank you so much!

**Comment by kenji-miyake** [comment]
@takayuki5168 Will you update the title?

**Comment by takayuki5168** [comment]
Ah right. Done.

**Review by takayuki5168 — APPROVED** [review]
LGTM

**Comment by kenji-miyake** [comment]
I think `*` is a bit unclear, so I fixed it again. :bow:

**Pull request merged**